src/components/views/MyCampaigns.jsx

```diff
@@ -234,12 +235,12 @@ class MyCampaigns extends Component {
                           </tbody>
                         </table>
 
-                        {campaigns.data.length > campaigns.itemsPerPage && (
+                        {campaigns.total > campaigns.limit && (
                           <center>
                             <Pagination
-                              activePage={campaigns.skipPages + 1}
-                              itemsCountPerPage={campaigns.itemsPerPage}
-                              totalItemsCount={campaigns.totalResults}
+                              activePage={campaigns.skip + 1}
```

Comment:
Needs to be divided by campaigns.limit . Also it is very strange to mix the service counting the records and the component counting records.